packages/firestore/test/lite/integration.test.ts

```diff
@@ -2116,6 +2117,45 @@ describe('countQuery()', () => {
     });
   });
 
+  it('count with filter and order by', () => {
+    return withTestCollectionAndInitialData(testDocs, async collection => {
+      const query1 = query(
+        collection,
+        where('author', '==', 'authorB'),
+        orderBy('title')
+      );
+      const countQuery_ = countQuery(query1);
+      const snapshot = await getAggregateFromServerDirect(countQuery_);
+      expect(snapshot.getCount()).to.equal(2);
+    });
+  });
+
+  it('count with order by and startAt, startAfter', () => {
```

Comment:
Please separate the tests for startAt and startAfter. Unit tests should just test one thing at a time. Same for endAt and endAfter.

go/unit-testing-practices#behavior-testing-examples